CMakeLists.txt

```diff
@@ -13,176 +13,164 @@
 #
 #   To build for Linux:
-#   1. Check out source hpcc-systems/wssql
-#   2. Check out pre-req source hpcc-systems/HPCC-Platform (check ~/wssql/platform-version-prereq.cmake for minimum version required)
-#   3. Setup Antlr and Antlr3c.
-#    a. "antlr-${ANTLR_VER}-complete.jar" and "antlr-runtime-${ANTLR_VER}.jar" are required.
-#    b. Build process looks for these files here: /usr/local/ANTLR/${ANTLR_VER} where ANTLR_VER = 3.4
+#   1.  Check out source hpcc-systems/wssql
+#   2.  Setup Antlr and Antlr3c.
+#    a. "antlr-${ANTLR_VER}-complete.jar" and "antlr-runtime-${ANTLR_VER}.jar"
+#       are required.
+#    b. Build process looks for these files here: /usr/local/ANTLR/${ANTLR_VER}
+#       where ANTLR_VER = 3.4
 #    c. "antlr-${ANTLR_VER}-complete.jar" is a build time dep
-#    e. "antlr-runtime-${ANTLR_VER}.jar" is a runtime dep (extract its contents in /usr/local/ANTLR/${ANTLR_VER})
-#    f. If building a distributable package, make sure to include ANTLR's license file: https://raw2.github.com/antlr/antlr4/master/LICENSE.txt in the ANTLR folder
-#   4. Create a build directory - preferably at the same level as the two source dirs "wssql and HPCC-Platform"
-#   5. cd to the build directory
-#   6a.To create makefiles to build native release version for local machine, run
-#       cmake ~/wssql
-#   6b.To create makefiles to build native debug version, run
-#       cmake -DCMAKE_BUILD_TYPE=Debug ~/wssql
-#   7. To build the makefiles just created above, run
-#       make
+#    e. "antlr-runtime-${ANTLR_VER}.jar" is a runtime dep(extract its contents
+#       in /usr/local/ANTLR/${ANTLR_VER})
+#    f. If building a distributable package, include ANTLR license file:
+#       https://raw2.github.com/antlr/antlr4/master/LICENSE.txt
+#   3.  Create build directory - preferably as a sibling of the source directory
+#   4.  cd to the build directory
+#   5a. To create makefiles to build native release version for local machine,
+#       run `cmake ~/wssql`
+#   5b. To create makefiles to build native debug version, run
+#       `cmake -DCMAKE_BUILD_TYPE=Debug ~/wssql`
+#   6.  To build the makefiles just created above, run `make`
 #
-#########################################################
+################################################################################
 
-PROJECT (hpccsystems-wssql)
+project(hpccsystems-wssql)
 
-if(UNIX AND NOT APPLE)
-    cmake_minimum_required(VERSION 3.3.2)
-else()
-    cmake_minimum_required(VERSION 2.8.11)
-endif()
+cmake_minimum_required(VERSION 3.3.2)
 
-set ( CMAKE_PROJECT_NAME "hpccsystems-wssql")
+set(CMAKE_PROJECT_NAME "hpccsystems-wssql")
 
-set (HPCC_WSSQL_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
-set (HPCC_SOURCE_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/..)
+set(HPCC_WSSQL_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
-IF ("${HPCC_SOURCE_DIR}" STREQUAL "")
-  SET (HPCC_SOURCE_DIR ${HPCC_SOURCE_ROOT_DIR}/HPCC-Platform)
-ENDIF()
+if("${HPCC_SOURCE_DIR}" STREQUAL "")
```

Comment:
can the build process be directed to use a local HPCC-Platform clone instead of the submodule?